### DISCUSSION TITLE
Generate a mixture property file（.fld file）

### DISCUSSION THREAD

**Discussion by 17pop** [usnistgov/REFPROP-issues#643] [General]
I would like to know about how to generate mixture medium file(type belongs to FLD file).My ultimate goal is to generate a mixture media RGP file via generator rGp.exe,and it is used for CFX simulation calculation.

**Comment by ianhbell** [comment]
I don't understand your question. Are you trying to make a .MIX file?

**Reply by 17pop** [reply]
I'm sorry for replying so late. Yes,I am trying to make the RGP file of the mixture.For examples,natural gas RGP files containing methane, ethane and propane.I really don't know how to use RGP GENERATOR to make it.

**Reply by ianhbell** [reply]
I have no idea what an RGP file is, sorry.

**Reply by 17pop** [reply]
It's nothing!Thank you for your reply! I will find some other methods on Internet
